### DISCUSSION TITLE
Some Album Covers Inverted in Main Window

### DISCUSSION THREAD

**Discussion by maxycatz** [jcorporation/myMPD#1364] [Q&A] [answered]
I have some album covers that show as inverted (negative) in the main window, but on the album view the cover appears correctly.

Any ideas why?

**Comment by jcorporation** [comment]
Please post a screenshot. Are this Images with transparency?

**Comment by maxycatz** [comment]
Just did a reinstall and it's not happening again.  If it happens again, will get screenshot.  No transparency; all images are jpeg.

**Comment by maxycatz** [comment]
Ah, it's here again.  There are a few inverted on the 'Albums' screen but I've attached two that show when an album is selected the screen shows the cover art correctly.
![shot-2024-10-29_22-18-31](https://github.com/user-attachments/assets/4a0d29cd-da1d-4d0a-9d02-a4fd912d741b)
![shot-2024-10-29_22-29-04](https://github.com/user-attachments/assets/1ab95a04-5617-4d70-8285-3298fcbdb101)
![shot-2024-10-29_22-29-31](https://github.com/user-attachments/assets/2d566d19-8eb1-4b20-aedd-45ce2923cd06)

**Reply by jcorporation** [reply]
Strange… Look into the developer tools in the network tab. How are the inverted Images are displayed there?

**Reply by maxycatz** [reply]
![shot-2024-10-29_22-55-33](https://github.com/user-attachments/assets/3d6594de-b3e0-4dc7-a507-bc3e154f0757)

**Comment by maxycatz** [comment]
This is the cover art that myMPD retrieves; there are no seperate thumbnail images:
![cover](https://github.com/user-attachments/assets/e7aa118c-8912-4ca5-9544-0a9397efa524)
![cover](https://github.com/user-attachments/assets/032dec07-e49e-4cb7-856c-9fdfe61e482e)

**Comment by jcorporation** [comment]
It seems to be a rendering issue in your browser. Have you tested an another browser?

**Comment by maxycatz** [comment, marked as answer]
Yes, Firefox and LibreWolf both of which were fine, but Chromium/Chrome has 'Install as app'.

But, the problem has been resolved!  It was Chromium's built in Dark Theme:
chrome://flags
Auto Dark Mode for Web Contents
Automatically render all web contents using a dark theme. – Mac, Windows, Linux, ChromeOS, Android, Lacros
#enable-force-dark

Looks like that mode has a problem with monotone/duotone type images.
